crates/core/src/subscription/module_subscription_manager.rs

```diff
@@ -252,7 +252,9 @@ impl SubscriptionManager {
             {
                 // Caller is not subscribed to any queries,
                 // but send a transaction update with an empty subscription update.
-                send_to_client(client, &event, SubscriptionUpdate::<DatabaseUpdate>::default());
+                let mut db_update = SubscriptionUpdate::<DatabaseUpdate>::default();
+                db_update.request_id = event.request_id;
+                send_to_client(client, &event, db_update);
```

Comment:
This was a bug fix, in this situation the request_id just wasn't getting sent back to the client who called the reducer 😦